speech/speech_samples.rb

```diff
@@ -77,85 +83,19 @@ def speech_async_recognize_gcs project_id:, storage_path:
   require "google/cloud/speech"
 
   speech = Google::Cloud::Speech.new project: project_id
-  audio  = speech.audio storage_path, encoding: :raw, sample_rate: 16000
+  audio  = speech.audio storage_path, encoding:    :raw,
```

Comment:
Why not place both positional and named parameters in one column? I don't mind the way it looks. I'm curious :)